HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,562 @@
+                },
+                "SubordinateResources": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "IfImplemented"
+                }
+            },
+            "ActionRequirements": {
+                "ResumeSubscription": {
+                    "ReadRequirement": "Recommended"
+                },
+                "SuspendSubscription": {
+                    "ReadRequirement": "Recommended"
+                }
+            }
+        },
+        "EventService": {
+            "PropertyRequirements": {
+                "ExcludeMessageId": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "IfImplemented"
+                },
+                "ExcludeRegistryPrefix": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "IfImplemented"
+                },
+                "IncludeOriginOfConditionSupported": {},
+                "RegistryPrefixes": {},
+                "ResourceTypes": {},
+                "ServerSentEventUri": {
+                    "ReadRequirement": "IfImplemented"
+                },
+                "ServiceEnabled": {},
+                "SubordinateResourcesSupported": {},
+                "Subscriptions": {}
+            },
+            "ActionRequirements": {
+                "TestEventSubscription": {
+                    "ReadRequirement": "Recommended"
+                },
+                "SubmitTestEvent": {
+                    "ReadRequirement": "Recommended"
+                }
+            }
+        },
+        "License": {
+            "ReadRequirement": "IfImplemented",
+            "PropertyRequirements": {
+                "AuthorizationScope": {},
+                "EntitlementId": {},
+                "ExpirationDate": {
+                    "ReadRequirement": "IfImplemented"
+                },
+                "InstallDate": {},
+                "LicenseInfoURI": {
+                    "ReadRequirement": "Recommended"
+                },
+                "LicenseOrigin": {},
+                "LicenseType": {},
+                "Manufacturer": {},
+                "Removable": {},
+                "SKU": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Status": {
+                    "PropertyRequirements": {
+                        "State": {
+                            "ReadRequirement": "Recommended"
+                        }
+                    }
+                }
+            }
+        },
+        "LicenseService": {
+            "ReadRequirement": "Recommended",
+            "Purpose": "For products that utilize software licensing, supporting the installation and updating of licenses through the Redfish interface is highly recommended.",
+            "PropertyRequirements": {
+                "Licenses": {}
+            },
+            "ActionRequirements": {
+                "Install": {
```

Comment:
Should we also make a requirement for "POST to the license collection"?

Reply:
At least as a recommendation, yes.

Reply:
I think we should consider requiring both. If there's a mix of support, then users will have to figure out when to host licenses on servers versus when to push directly; some users may not have much of a choice in their environment.